Cargo.toml

```diff
@@ -31,15 +31,19 @@ serde = { version = "1.0.106", features = ["derive"] }
 serde_json = "1.0.51"
 thiserror = "1.0.14"
 void = "1.0.2"
+ipfs-unixfs = { path = "unixfs" }
+async-stream = "0.2.1"
 
 [build-dependencies]
 prost-build = "0.6.1"
 
 [dev-dependencies]
 env_logger = "0.7.1"
+sha2 = "*" # this is dependent on by multihash but it doesn't expose the Digest functionality needed in test
+hex-literal = "*"
```

Comment:
I know this is just a dev dependency, but it might be better to specify the versions here in order to avoid issues when the crates are updated.

Reply:
I lazyily always start with star as I know it tends to get the version already included in the workspace but of course always forget to change this.